### ISSUE TITLE
Stack Overflow using defaultOn=false System Property

### ISSUE THREAD

**Issue by cbegin** [adangel/liquibase-percona#26]
Liquibase-Percona: 1.5.0
Liquibase: 3.5.3
pt-online-schema-change: 3.0.5
MariaDB Java Connector: 2.3.0
Docker Image: openjdk:8u131-jre-alpine

Setting the defaultOn system property results in a stack overflow error, seemingly from the toString() function called in the debug line here:

https://github.com/adangel/liquibase-percona/blob/liquibase-percona-1.5.0/src/main/java/liquibase/ext/percona/PerconaChangeUtil.java#L76

```
JAVA_OPTS="-Dliquibase.percona.defaultOn=false"
```

```
 liquibase.exception.UnexpectedLiquibaseException: liquibase.exception.UnexpectedLiquibaseException: liquibase.exception.UnexpectedLiquibaseException: liquibase.exception.UnexpectedLiquibaseException: liquibase.exception.UnexpectedLiquibaseException: java.lang.StackOverflowError
        at liquibase.change.AbstractChange.createChangeMetaData(AbstractChange.java:93)
        at liquibase.change.ChangeFactory.getChangeMetaData(ChangeFactory.java:101)
        at liquibase.change.AbstractChange.toString(AbstractChange.java:530)
        at liquibase.ext.percona.PerconaChangeUtil.generateStatements(PerconaChangeUtil.java:76)
        at liquibase.ext.percona.PerconaAddColumnChange.generateStatements(PerconaAddColumnChange.java:58)
        at liquibase.change.AbstractChange.generateStatementsVolatile(AbstractChange.java:261)
        at liquibase.change.AbstractChange.supports(AbstractChange.java:301)
        at liquibase.change.ChangeParameterMetaData.analyzeSupportedDatabases(ChangeParameterMetaData.java:95)
        at liquibase.change.ChangeParameterMetaData.<init>(ChangeParameterMetaData.java:78)
        at liquibase.change.AbstractChange.createChangeParameterMetadata(AbstractChange.java:142)
        ... 194 more
```

**Comment by adangel** [comment]
Thanks for reporting - the bug is fixed now.
To use the snapshot version, see https://github.com/adangel/liquibase-percona#using-snapshots

**Comment by cbegin** [comment]
Cool, thanks. We will give it a try this week!

**Comment by adangel** [comment]
@cbegin I've just released 1.5.1: https://github.com/adangel/liquibase-percona/releases/tag/liquibase-percona-1.5.1
If you find any problems, please open a new issue. Thanks for testing!
